.evergreen/run-custom-csfle-tests.sh

```diff
@@ -20,49 +20,32 @@ set -o errexit  # Exit the script with error if any of the commands fail
 # Get access to the AWS temporary credentials:
 echo "adding temporary AWS credentials to environment"
 # CSFLE_AWS_TEMP_ACCESS_KEY_ID, CSFLE_AWS_TEMP_SECRET_ACCESS_KEY, CSFLE_AWS_TEMP_SESSION_TOKEN
-. "$DRIVERS_TOOLS"/.evergreen/csfle/set-temp-creds.sh
-
-ABS_PATH_TO_PATCH=$(pwd)
+source "$DRIVERS_TOOLS"/.evergreen/csfle/set-temp-creds.sh
 
 # Environment Variables:
 # CSFLE_GIT_REF - set the git reference to checkout for a custom CSFLE version
-# CDRIVER_GIT_REF - set the git reference to checkout for a custom CDRIVER version (this is for libbson)
-CSFLE_GIT_REF=${CSFLE_GIT_REF:-master}
-CDRIVER_GIT_REF=${CDRIVER_GIT_REF:-1.17.6}
+CSFLE_GIT_REF=${CSFLE_GIT_REF:-main}
```

Comment:
Any reason you didn't just re-use the tooling we have in main? https://github.com/mongodb/node-mongodb-native/blob/main/.evergreen/install-mongodb-client-encryption.sh

Reply:
Honestly did not even think to 😅 Happy to update